eng/versioning/external_dependencies.txt

```diff
@@ -315,3 +315,6 @@ storage_com.microsoft.azure:azure-storage;8.4.0
 
 # sdk\spring\azure-spring-boot\pom.xml
 spring_com.microsoft.azure:azure;1.34.0
+
+# sdk\spring\azure-spring-boot-starter-servicebus-jms\pom.xml
+com.microsoft.azure:azure-servicebus-jms;0.0.5
```

Comment:
If the combination is not prefixed by `spring_`, we could place it above with all other external dependencies, instead of having a separate block for it.